src/examples/design/icons/SvgSearch.tsx

```diff
@@ -26,7 +26,7 @@ const StyledSvgWrapper = styled.div<{ isAnswerBot?: boolean }>`
   display: flex;
   align-items: center;
   margin: 0 0 ${p => p.theme.space.sm};
-  fill: ${p => p.isAnswerBot && PALETTE.kale[700]};
+  fill: ${p => p.isAnswerBot && getColorV8('kale', 700, p.theme)};
```

Comment:
```suggestion
  fill: ${p => p.isAnswerBot && getColor({ theme: p.theme, hue: 'kale', shade: 900 })}};
```

Reply:
I think this was intentionally left to `getColorV8` but I could be mistaken (re: PR description).

Reply:
Ha! I started reviewing before the description was finalized. I'll let @jzempel make that call. 😄